app/main/model/audio.py

```diff
@@ -22,11 +24,27 @@ class Audio(db.Model):
     db.Index('ix_audios_context', context, postgresql_using='gin'),
   )
 
+  @property
+  def canned_response(self):
```

Comment:
why is this a "canned_response"? what does it mean? (looks like it is still computing a fingerprint?)

Reply:
A canned response is just returning the existing value stored in the DB - if we already have a value for a URL, we are making the assumption that the fingerprint is still valid (which, if we cut the other way, we sign up for re-encoding lots of things over and over needlessly)

Reply:
maybe existing_response?